en-US/firefox/browsers.lang

```diff
@@ -0,0 +1,90 @@
+## NOTE: Demo page at https://www-demo1.allizom.org/firefox/browsers/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/browsers/
+
+
+# HTML page title
+;Get the browsers that put your privacy first — and always have
+Get the browsers that put your privacy first — and always have
+
+
+# HTML page description
+;Get the privacy you deserve. Enhanced Tracking Protection is automatic in every Firefox browser.
+Get the privacy you deserve. Enhanced Tracking Protection is automatic in every Firefox browser.
+
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+;Get the <strong>browsers</strong> that put your privacy first — and always have
+Get the <strong>browsers</strong> that put your privacy first — and always have
+
+
+;Desktop
+Desktop
+
+
+;Seriously private browsing. Firefox automatically blocks 2000+ online trackers from collecting information about what you do online.
+Seriously private browsing. Firefox automatically blocks 2000+ online trackers from collecting information about what you do online.
+
+
+;Download for Desktop
+Download for Desktop
+
+
+;Learn more
+Learn more
+
+
+;Mobile
+Mobile
+
+
+;Take the same level of privacy — plus your passwords, search history, open tabs and more — with you wherever you go.
+Take the same level of privacy — plus your passwords, search history, open tabs and more — with you wherever you go.
+
+
+;Download on the App Store
+Download on the App Store
+
+
+;Download for Mobile
+Download for Mobile
+
+
+;Send me a link
+Send me a link
+
+
+;Enterprise
+Enterprise
+
+
+;Get unmatched data protection with support cycles tailored to suit your company’s needs.
+Get unmatched data protection with support cycles tailored to suit your company’s needs.
+
+
+;Enterprise packages
+Enterprise packages
+
+
+;Reality
+Reality
+
+
+;Go beyond two dimensions and enjoy the best immersive content from around the web.
+Go beyond two dimensions and enjoy the best immersive content from around the web.
+
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+;Build sites and refine your code with Firefox <strong>DevTools</strong>
+Build sites and refine your code with Firefox <strong>DevTools</strong>
+
+
+;Learn more about DevTools
+Learn more about DevTools
+
+
+;Join Firefox and get the most out of every product — across every device.
+Join Firefox and get the most out of every product — across every device.
+
+
+;Already have an account? <a %(fxa_attr)s>Sign In</a> or <a %(accounts_attr)s>learn more</a> about Joining Firefox</p>
```

Comment:
1). why is Joining Firefox with a capital J.
2). Do we need </p>?

Reply:
Good catches. I will fix those things here & in bedrock.

Reply:
Corresponding bugfix PR in bedrock - https://github.com/mozilla/bedrock/pull/8493